src/components/atoms/FormattedInput/FormattedInput.tsx

```diff
@@ -104,7 +106,7 @@ const FormattedInput = forwardRef<HTMLInputElement, FormattedInputProps>(
     }, [currentParsedValue, defaultValue, emptyValue, format, parse, value]);
 
     return (
-      <div className={styles.root}>
+      <div className={clsx(styles.root, className)}>
```

Comment:
if we are passing the className to the wrapper, we'll have to pass an innerClassName to the TextInput, because we wont be able to modify the parent div for TextInput otherwise.